CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# How to Contribute
+
+We greatly appreciate contributions to the vscode-java-dependency project. Your efforts help us maintain and improve this extension. To ensure a smooth contribution process, please follow these guidelines.
+
+## Setup Guide
+
+To set up the vscode-java-dependency project, follow these steps:
+
+1. **Build the Server JAR**:
+   - The server JAR (Java application) is located in the [jdtls.ext](./jdtls.ext) directory.
+   - Run the following command to build the server:
+     ```shell
+     npm run build-server
+     ```
+
+2. **Install Dependencies**:
+   - Execute the following command to install the necessary dependencies:
+     ```shell
+     npm install
+     ```
+
+3. **Run the Extension**:
+   - Open the "Run and Debug" view in Visual Studio Code.
+   - Run the "Run Extension" task.
+
+4. **Attach to Plugin**:
+   - Open the "Run and Debug" view in Visual Studio Code.
+   - Run the "Attach to Plugin" task.
```

Comment:
A prerequisite of this launch configuration is that the extension is activated, in other words, the java process is launched. Let's add this information as well in the document.

Reply:
Updated.